deploy/registration-service/registration-service.yaml

```diff
@@ -27,6 +27,7 @@ objects:
           - get
           - update
           - list
+          - watch
```

Comment:
This change is not actually related to this issue. It is related to proxy caching work but just leaving it here since it's a small change.

Reply:
Just keep in mind that we need to keep it consistent with https://github.com/codeready-toolchain/host-operator/blob/master/deploy/registration-service/registration-service.yaml#L25-L29

Reply:
I guess you meant https://github.com/codeready-toolchain/registration-service/blob/master/deploy/registration-service.yaml#L25-L29  

Good point, I totally forgot about that! Will create a PR for that.

Reply:
https://github.com/codeready-toolchain/registration-service/pull/296

Reply:
Yes! :)